clang/lib/3C/Constraints.cpp

```diff
@@ -121,6 +123,17 @@ bool Constraints::addConstraint(Constraint *C) {
     return true;
   }
 
+  // If the constraint being added is due to unwritability,
+  // propagate this reason to the existing constraint.
+  // This way we always prioritize the unwritability as the reason
+  // for wildness.
```

Comment:
Also link directly to https://github.com/correctcomputation/checkedc-clang/issues/664, which explains why we prioritize unwritability in the absence of support for reporting multiple reasons?